packages/terra-functional-testing/src/commands/axe/run.js

```diff
@@ -1,40 +1,51 @@
 /* global browser, axe */
 const injectAxe = require('./inject');
 
+/**
+ * Global rule overrides.
+ * Rules modified here will be applied globally for all tests.
+ */
+const GLOBAL_RULE_OVERRIDES = {
+  /**
+   * This rule was introduced in axe-core v3.3 and causes failures in many Terra components.
+   * The solution to address this failure vary by component. It is being disabled until a solution is identified in the future.
+   *
+   * Reference: https://github.com/cerner/terra-framework/issues/991
+   */
+  'scrollable-region-focusable': { enabled: false },
+};
+
 /**
  * Executes axe on the browser.
- * @param {Object} overrides - The axe options.
- * @param {Array} overrides.rules - The rule overrides.
+ * @param {Object} options - The axe options.
+ * @param {Object} options.rules - The rule overrides.
  */
-const runAxe = (overrides = {}) => {
+const runAxe = (options = {}) => {
   // Extract the axe options for the Terra service from the global browser object.
-  const [, options = {}] = browser.options.services.find(([service]) => (
+  const [, serviceOptions = {}] = browser.options.services.find(([service]) => (
     typeof service === 'function' && service.name === 'TerraService'
   ));
 
-  const { axe: axeOptions } = options;
+  const { axe: axeServiceOptions = {} } = serviceOptions;
+
   const isAxeUnavailable = browser.execute(() => window.axe === undefined);
 
   // Inject axe-core onto the page if it has not already been initialized.
   if (isAxeUnavailable) {
-    injectAxe(axeOptions);
-  }
+    const globalRules = { ...GLOBAL_RULE_OVERRIDES, ...axeServiceOptions.rules };
 
-  /**
-    * This rule was introduced in axe-core v3.3 and causes failures in many Terra components.
-    * The solution to address this failure vary by component. It is being disabled until a solution is identified in the future.
-    *
-    * Reference: https://github.com/cerner/terra-framework/issues/991
-    */
-  const ruleOverrides = {
-    'scrollable-region-focusable': { enabled: false },
-  };
+    // The axe.configure API requires the rules to be an array of objects.
+    const globalRulesArray = Object.keys(globalRules).map((rule) => (
+      { ...globalRules[rule], id: rule }
+    ));
+
+    injectAxe({ ...axeServiceOptions, rules: globalRulesArray });
+  }
 
-  // Merge the global rules and overrides together.
   const rules = {
-    ...ruleOverrides,
-    ...axeOptions && axeOptions.rules,
-    ...overrides.rules,
+    ...GLOBAL_RULE_OVERRIDES,
+    ...axeServiceOptions.rules,
```

Comment:
I thought we decided to drop this Service Level configuration option in our sync last Wednesday? So no axe configuration when initialized the TerraService or flat on the wdio object, but instead allow per test overrides and then the global be defined like you have in `GLOBAL_RULE_OVERRIDES`.

Reply:
I investigated removing the axe configuration from the service options, but we're going to need it to dynamically flex the color contrast rule while in lowlight theme. Either to disable it or adjust the contrast ratio.

Here's how we dynamically flex the rules today:

https://github.com/cerner/terra-toolkit-boneyard/blob/main/config/wdio/wdio.conf.js#L102

Here's an example of what removing the axe option looks like:

https://github.com/cerner/terra-toolkit/compare/remove-terra-service-axe-option

We might be able to flex the option within the service, but until we have a solution we'll need to keep it as a service option. These changes are just cleaning up the file, they're not introducing any new functionality

Reply:
I assumed we'd be flex that in the service like how we are overriding the scroll rule. It's always be disabled (or warn) for the lowlight theme.

Reply:
The scroll rule is static and never changes, but the color contrast rule would be dynamic based on the theme.

Removing the axe option from the service is outside the scope of this PR but I'll do additional investigation into removing the axe options while still dynamically flexing rules based on the theme. If I come up with a solution I'll make a separate PR. Removing the axe options from the service doesn't impact the functionality of the accessibility reporter

Reply:
I've created a PR to address these changes, I'd like to move axe configuration discussions onto that PR:

https://github.com/cerner/terra-toolkit/pull/466

I've removed all changes not directly associated to the accessibility reporter from this PR